.nais/naiserator-dev.yaml

```diff
@@ -67,6 +67,8 @@ spec:
         diskAutoresize: true
   envFrom:
     - secret: ispersonoppgave-serviceuser
+  kafka:
```

Comment:
Erstatter dette eksisterende Kafka-topics? I så fall kan kanskje env-variable for Kafka fjernes?